.gitignore

```diff
@@ -6,6 +6,7 @@
 *.pot
 *.py[co]
 .tox/
+.idea/
```

Comment:
Can you remove this change? I know you probably wanted to ignore your pycharm config files or something like this but it's important to keep the PRs to only fix 1 thing at a time :+1: